src/view/configuration/edgeConfigurationsSection.jsx

```diff
@@ -301,23 +301,31 @@ const getSelectInputMethodStateForNewInstance = async ({
   // particularly so when a user clicks the Add Instance button,
   // they don't have to wait while the instance is created.
   if (!firstPageOfSandboxes) {
-    ({ results: firstPageOfSandboxes } = await fetchSandboxes({
-      orgId,
-      imsAccess
-    }));
+    try {
+      ({ results: firstPageOfSandboxes } = await fetchSandboxes({
+        orgId,
+        imsAccess
+      }));
+    } catch (error) {
+      context.current.fetchSandboxError = true;
+    }
   }
 
   context.current.sandboxes = firstPageOfSandboxes;
 
   // checking if this is a organization with one sandbox ( default sandbox )
   if (firstPageOfSandboxes.length === 1) {
```

Comment:
If there is a fetchSandboxError won't firstPageOfSandboxes be undefined? Checking the length of undefined would cause an error.

Reply:
yes, thank you